packages/cloud-cognitive/src/components/NonLinearReading/_non-linear-reading.scss

```diff
@@ -8,31 +8,35 @@
 // Standard imports.
 @import '../../global/styles/project-settings';
 
-@mixin term-closed-defaults {
+@mixin keyword-closed-defaults {
+  height: rem(20px);
+  padding-top: 0;
   // stylelint-disable-next-line carbon/layout-token-use
-  padding: rem(1px) $spacing-03 rem(1px) $spacing-03;
-  border: none;
+  padding-right: rem(3px);
+  border-width: rem(1px);
+  border-style: solid;
   border-radius: $spacing-04;
-  background-color: $interactive-02;
   white-space: nowrap;
 
   &:hover {
-    background-color: $hover-secondary;
     cursor: pointer;
   }
-}
-@mixin term-open-defaults {
-  @include term-closed-defaults();
-
-  padding-right: 0;
 
+  // The "up" chevron
   svg {
     // stylelint-disable-next-line carbon/layout-token-use
-    padding-top: rem(1px);
-    margin: 0 $spacing-02 0 $spacing-01;
+    margin: rem(1px) 0 0 0;
     vertical-align: text-top;
   }
 }
+@mixin keyword-open-defaults {
+  @include keyword-closed-defaults();
+
+  // The "up" chevron, flipped
+  svg {
+    transform: rotate(180deg);
```

Comment:
Would recommend adding a transition for rotation in the future as an easy win.

Reply:
Animations were specified by the designers for other elements, but not for the chevron.